hud/src/commonMain/kotlin/HUD.kt

```diff
@@ -26,61 +16,61 @@ Copyright 2019 Splendo Consulting B.V. The Netherlands
 
 */
 
+package com.splendo.kaluga.hud
+
+import co.touchlab.stately.concurrency.Lock
+import co.touchlab.stately.concurrency.withLock
+import com.splendo.kaluga.base.MainQueueDispatcher
+import com.splendo.kaluga.base.MultiplatformMainScope
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
 /**
- * Interface that defines loading indicator class, which can be shown or dismissed
+ * Style of the Loading Indicator
  */
-interface HUD : CoroutineScope {
-
-    /**
-     * Style of the Loading Indicator
-     */
-    enum class Style(val value: Int) {
-        /** System appearance */
-        SYSTEM(0),
-        /** Custom appearance */
-        CUSTOM(1);
-
-        companion object {
-            fun valueOf(value: Int) = values().first { it.value == value }
-        }
+enum class HUDStyle(val value: Int) {
```

Comment:
I know it's old code but this can just use cardinal values and `com.splendo.kaluga.utils` `byOrdinalOrDefault`